formats/polus-ome-converter-plugin/src/main.py

```diff
@@ -32,6 +32,10 @@ def image_converter(inp_image, fileExtension, out_dir):
 
                 # Loop through channels
                 for c in range(br.C):
+
+                    if fileExtension is None:
+                        FILE_EXT = FILE_EXT
+
                     
                     FILE_EXT = fileExtension
```

Comment:
I think you're missing some logic in here. You should also move this logic out of the for loop.

You should probably replace these lines with:
```python
FILE_EXT = FILE_EXT if fileExtension is None else fileExtension
```